src/net/sourceforge/kolmafia/FamiliarData.java

```diff
@@ -1187,6 +1187,13 @@ private static void loadDropFamiliars() {
             "grubby wool",
             "_grubbyWoolDrops",
             -1));
+    DROP_FAMILIARS.add(
+        new DropInfo(
+            FamiliarPool.JILL_OF_ALL_TRADES,
+            ItemPool.MAP_TO_A_CANDY_RICH_BLOCK,
+            "maps",
+            "_mapToACandyRichBlockDrops",
+            3));
```

Comment:
Since you are not sure about a cap of 3 would it be better to leave this uncapped until there is come certainty?

Not intended to block.

Reply:
yeah probably